lldb/source/Plugins/Language/Swift/SwiftUnsafeTypes.cpp

```diff
@@ -64,11 +75,12 @@ lldb::addr_t SwiftUnsafeType::GetAddress(llvm::StringRef child_name) {
     return false;
   }
 
-  auto type_system = llvm::dyn_cast<SwiftASTContext>(type.GetTypeSystem());
+  SwiftASTContext *type_system = GetSwiftASTContextTypeSystem(type);
```

Comment:
Now this should just be 

`auto *typesystem = dyn_cast_or_null<TypeSystemSwift>(type.GetTypeSystem())`

and GetSwiftASTContextTypeSystem can be removed, right?

Reply:
Indeed!